docs/docs/get-started/setup-lightdash/install-lightdash.mdx

```diff
@@ -0,0 +1,173 @@
+---
+sidebar_position: 1
+sidebar_label: Install Lightdash
+---
+import TOCInline from '@theme/TOCInline';
+
+# Install Lightdash
+
+<TOCInline toc={toc} />
+
+## Deploy locally with our installation script
+
+This is the recommended way to install and run Lightdash in your local machine.
+
+:::caution Production deployments
+
+For production deployments check the [Deploy manually](#deploy-manually) instructions.
+
+:::
+
+:::info
+
+This script is going to install docker and docker compose. If you want do use a docker alternative, see [Deploy manually](#deploy-manually) instructions.
+
+:::
+
+#### 1. Clone the Lightdash repository
+
+Clone the Lightdash code to your local machine. This will create a new directory called `./lightdash` (the Lightdash directory).
+
+```bash
+# Clone the Lightdash repo
+git clone https://github.com/lightdash/lightdash
+cd lightdash
+# A new directory called "lightdash" should appear
+```
+
+#### 2. Install & Launch Lightdash
+
+:::info
+
+If you're using a local dbt project, during the cli, select the option **"with local dbt"**
+and during the project creation keep the **Project directory** value as **"/usr/app/dbt"**.
+
+:::
+
+```bash
+./install.sh
+# follow cli instructions
+```
+
+----
+
+## Deploy to Heroku
+
+Click to deploy the latest version of Lightdash to heroku. This provides a secure, production-ready deployment.
```

Comment:
What's the difference between a production-ready deployment and a production deployment? Like, if I wanted to share my Lightdash project with my team, could I use the one-click-deploy to Heroku? Or what would I have to do to get it production-ready?

Reply:
> What's the difference between a production-ready deployment and a production deployment?

There isn't any difference. I can remove the `-ready` part.

> Like, if I wanted to share my Lightdash project with my team, could I use the one-click-deploy to Heroku? 

yes